packages/view-util/lib/dimensions.js

```diff
@@ -51,3 +51,19 @@ export async function populateDimensionIdentifiers(pointer, metaLookup) {
       }
     })
 }
+
+export function setViewAs(view) {
```

Comment:
I would add some cues in the function name,  'setViewAsProp' as it can be read as 'set view as view'